src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence.mdx

```diff
@@ -12,36 +12,103 @@ redirects:
+
+If you need cross-account collaboration, you can add multiple accounts into a single custom environment. You should only do this if you need to track cross-account data.
+
+By default, each account is in its own environment if it was created after November 2021. This is the recommended setting. 
+
+### What's stored in the environment? [#environment-detail]
+
+<img
+  title="The incident intelligence environment."
+  alt="A screenshot of the incident intelligence environment dropdown."
+  src={environment}
+/>
+
+<figcaption>
+  Use the environment dropdown to see your cross-account applied intelligence activity.
+</figcaption>
+
+* Your workflows configuration
+* Your decisions configuration
+* Your incidents
+* Issues created from your incidents
 
-To set up an environment in incident intelligence, you need an administrator to select a New Relic account for it. This account should be the one your team is using.
+Workflow enricher queries are executed at the environment account level.
 
-* **Who sets the environment?** Only administrators, and only for accounts where they have admin privileges.
-* **Can administrators set more than one environment?** They can set one environment per parent account and its child accounts. More than one can be set if an administrator has privileges for more than one parent account.
+### What's stored in the individual account? [#account-detail]
 
-<Callout variant="tip">
-  Incident intelligence is a cross-account product. This means you can send in data from any New Relic account or external source to correlate events.
+<img
+  title="The incident intelligence individual account."
+  alt="A screenshot of the incident intelligence individual account dropdown."
+  src={accountTeam}
+/>
+
+<figcaption>
+  Use the individual account dropdown to see policies, conditions, and destinations related to your account.
+</figcaption>
+
+* Your policies and conditions
+* Your destinations
+
+When moving individual accounts from one environment to another, the configurations aren't deleted, but they may not be available in the new environment.
+
+<Callout variant="important">
+  To view an issue, a user must be authorized for read access to the incident intelligence environment.
+
+To view an incident's details, a user must be authorized for read access to the account where the incident originated.
+
+We recommend giving read access to all of your users in each incident intelligence environment.
 </Callout>
 
+### Create a custom environment [#create-environment]
+
+As an administrator, complete these steps to create a new custom environment for the account your team is using. If you're not an New Relic administrator, you won't be able to complete these steps.
+
+1. Create a [user API key](https://one.newrelic.com/api-keys).
+2. Go to [NerdGraph, our GraphiQL explorer](https://api.newrelic.com/graphiql). (If you're on an EU datacenter, use [EU NerdGraph](https://api.eu.newrelic.com/graphiql).
```

Comment:
the explorer is called 'the NerdGraph explorer'. in other words, the explorer is different than Nerdgraph (they can use nerdgraph without using the explorer, in other words).